PriceCalculatorKata/Product.cs

```diff
@@ -6,6 +6,7 @@ namespace PriceCalculatorKata
 {
     public class Product
     {
+        public static float discount = 0;
```

Comment:
it's better to use properties for public values to encapsulate it with a getter and a setter.